multiaccounts/accounts/database.go

```diff
@@ -974,3 +998,29 @@ func (db *Database) UpdateAccountToFullyOperable(keyUID string, address types.Ad
 	_, err = tx.Exec(`UPDATE keypairs_accounts SET operable = ?	WHERE address = ?`, AccountFullyOperable, address)
 	return err
 }
+
+func (db *Database) UpdateAccountPosition(address types.Address, newPosition uint64) error {
+	tx, err := db.db.Begin()
+	defer func() {
+		if err == nil {
+			err = tx.Commit()
+			return
+		}
+		_ = tx.Rollback()
+	}()
+
+	if err != nil {
+		return err
+	}
+
+	_, err = tx.Exec("UPDATE keypairs_accounts SET position = position + 1 WHERE position >= ?", newPosition)
```

Comment:
If we have accounts [1, 2, 3, 4, 5], moving 2 to 4 will result in [1, 3, 4, 5, 6] not keeping the strict position index. Is this intended?

Reply:
Yeah, I didn't think it was important to keep index as 1,2,3,4......
As we won't have billions of accounts anyway

Reply:
That is why I added the previous comment to create this awareness. Otherwise, the `positon` column name might lead to an assumption that this is a real position and not just an order hint and can be used in other operations.

Reply:
Yes fine adding this comment, also not that if u delete an account i do not update the position too, cause it doesn't have to be